### PR TITLE
Propose adding Jason Sherman usingtechnology as a Maintainer

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
@hyperledger/aries-cloudagent-python-committers @hyperledger/aries-admins -- we need multiple approvals on this PR to merge it.  We will discuss this on the upcoming ACA-Pug meeting --- June 13, 2023.

**Review by andrewwhitehead — APPROVED** [review]

**Review by dbluhm — APPROVED** [review]

**Review by dhh1128 — APPROVED** [review]

**Review by TimoGlastra — APPROVED** [review]

**Review by ianco — APPROVED** [review]

**Comment by ianco** [comment]
> This PR also corrects the Maintainers list by adding @ianco. As he is on the "Committers" team, he is by definition a Maintainer. I missed including him in PR #2258. @ianco recently left the BC Gov Digital Trust team ☹️ . I'm not sure if he is willing to continue to be a Maintainer, or if he should move to be a "Maintainer Emeritus". We'd be happy for him to stay as a Maintainer!

Happy to remain as maintainer!

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**
